ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/[slug]/(dashboard)/DashboardTimeRangeFilter.tsx

```diff
@@ -98,12 +106,7 @@ export default function DashboardTimeRangeFilter({
     query: GetBillingPlanDocument,
   });
 
-  // Since "features" is a map, we can't be 100% sure that there's a log
-  // retention value. So default to 7 days.
-  let logRetention = 7;
-  if (typeof data?.account.plan?.features.log_retention === 'number') {
-    logRetention = data.account.plan.features.log_retention;
-  }
+  const logRetention = data?.account.plan?.entitlements.history.limit || 7;
```

Comment:
```suggestion
  const logRetention = data?.account.entitlements.history.limit || 7;
```
We want to use the `Account.entitlements` since `BillingPlan.entitlements` doesn't consider account overrides. Also, if we ever create a history addon then it'll reflect in `Account.entitlements` and not `BillingPlan.entitlements`

Reply:
The reason I was doing it is because I didn't wanna mess with those old time filters, that map the upgrades to specific plans. That's the reason for my comment in the PR about centralizing the logic, so we can get rid of this old code. To do what you are suggesting I need to rip that apart, which is fine but additional scope
<img width="326" alt="Screenshot 2024-12-10 at 18 32 51" src="https://github.com/user-attachments/assets/87e7c4f1-b1c5-4267-92df-589dbc49e780">